harvester/src/test/java/com/github/reflectoring/infiniboard/harvester/scheduling/config/UpdatePluginConfigJobTest.java

```diff
@@ -82,8 +82,8 @@ public void executeInternalSchedulesAllGivenJobs()
         ArrayList<WidgetConfig> widgetConfigs = new ArrayList<>();
         when(repository.findAll()).thenReturn(widgetConfigs);
 
-        widgetConfigs.add(createWidgetConfig("firstWidget", "one", "two", "three"));
-        widgetConfigs.add(createWidgetConfig("secondWidget", "alpha", "beta", "gamma"));
+        widgetConfigs.add(createWidgetConfig("firstWidget", "widgetType", "one", "two", "three"));
+        widgetConfigs.add(createWidgetConfig("secondWidget", "widgetType", "alpha", "beta", "gamma"));
```

Comment:
Use values as close to production as possible for testing like `platform-status`. This way we validate, that we do not run into issues on characters like `-` used in the real widget types.